pkg/datarepair/repairer/repairer.go

```diff
@@ -27,8 +33,9 @@ type Repairer interface {
 
 // Config contains configurable values for repairer
 type Config struct {
-	// queueAddress string `help:"data repair queue address" default:"localhost:7779"`
-	maxRepair int `help:"maximum segments that can be repaired concurrently" default:"100"`
+	queueAddress string `help:"data repair queue address" default:"localhost:6379"`
+	queuePass    string `help:"data repair queue password" default:""`
```

Comment:
Maybe pass in similarly how overlay and pointerdb does it. That way you don't have to have separate password field and can pass in different implementations (later). Then you can specify it as `redis://localhost:6379?db=5&password=123`.

See:
https://github.com/storj/storj/blob/b6b6111173e524ec55d3abc699c37ef816d51786/pkg/pointerdb/config.go#L26
https://github.com/storj/storj/blob/b6b6111173e524ec55d3abc699c37ef816d51786/pkg/overlay/config.go#L46